### PR TITLE
upgrade to lodash4, move lodash to deps instead of peerDeps

### LINKED ISSUE #16: Please update lodash to 4.x
Lodash 4.x support IE9+ as well.

### PR DESCRIPTION
**Pull request by bywo**
resolves #16 #20 

@rafaelchiti can I get a sanity-check on this one?

### CODE REVIEW

File: src/update_engine_ie9.jsx [hunk 1]
```diff
@@ -36,36 +38,37 @@ export function deregister(component) {
...
+  invokeMap(components, '_applyWidths');
 
   // apply heights now that widths have been set
-  _.invoke(components, '_measureItemHeights');
-  _.invoke(components, '_applyFlexHeights');
+  invokeMap(components, '_measureItemHeights');
+  invokeMap(components, '_applyFlexHeights');
 
-  // NOTE: each container must be set sequentially instead of batched because child Layout heights can depend on
+  // NOTE: each contai
```
Comment (gianu) [inline src/update_engine_ie9.jsx]: remove this line break

**Comment by gianu** [comment]
besides the small comment, :+1:

File: src/horizontal_ie9.jsx [hunk 2]
```diff
@@ -1,4 +1,7 @@
-import _ from 'lodash';
+import extend from 'lodash/extend';
+import range from 'lodash/range';
+import invokeMap from 'lodash/invokeMap';
+import sum from 'lodash/sum';
```
Comment (rafaelchiti) [inline src/horizontal_ie9.jsx:4]: does this prevent the whole lodash library to get bundled?

Comment (bywo) [inline src/horizontal_ie9.jsx:4]: i believe so!

Comment (rafaelchiti) [inline src/horizontal_ie9.jsx:4]: hehe yeah seems like, just npm installed it and is very well fractioned, was going to suggest to use the `per method` distributed version but this is the same and easier, cool :) 👍

Comment (pechitook) [reply src/horizontal_ie9.jsx:4]: How would one test _properly_ if this is the case? I have `webpack-dashboard` and it claims the whole lodash library is being bundled.

**Pull request merged**